examples/miscellaneous/bayesBenchmark/directCompare3D.m

```diff
@@ -1,135 +1,127 @@
-%Comparison of MCMC, NS and direct calculation.
+% This example compares three Bayesian posteriors for a low-dimensional
+% example: a posterior generated by DREAM, one generated by NS, and
+% one calculated directly.
+% 
+% In this 2D case, we calculate the posterior for a fit over substrate
```

Comment:
Please update comment, this comment is exactly the same as the one in `directCompare2D `